templates/oe_contact/oe-contact--default.html.twig

```diff
@@ -0,0 +1,27 @@
+{#
+/**
+ * @file
+ * Theme implementation to present a contact entity.
+ *
+ * @see ./modules/contrib/oe_content_entity/modules/oe_content_entity_contact/templates/oe-contact.html.twig
+ */
+#}
+<div class="ecl-row">
+  <div class="ecl-col-12">
+    <h3 class="ecl-u-type-heading-3 ecl-u-type-color-black ecl-u-mt-none ecl-u-mb-none">{{ content.name }}</h3>
+  </div>
+  <div class="ecl-col-12 ecl-col-md-6 ecl-u-pb-m">
+    {% if entity.oe_body is not empty %}
+      <div class="ecl-editor">
+        {{ content.oe_body }}
+      </div>
+    {% endif %}
+    {{ content.group_contact_information }}
+    {{ content.oe_press_contact_url }}
+  </div>
+  {% if entity.oe_image is not empty %}
+    <div class="ecl-col-12 ecl-col-md-5">
```

Comment:
Wrapper class is `ecl-col-md-5` to set width for image. Now it is 445px, but on the mockup - 450px (5px difference). 
Width wrapper class `ecl-col-md-6` image's width will be 540px.

<img width="1065" alt="Screenshot 2020-08-04 at 10 57 16" src="https://user-images.githubusercontent.com/1183951/89268840-bf5d4300-d641-11ea-88b3-6d7771513e3a.png">

Reply:
That's OK, we will let business decide on UAT.